test/integration/targets/ios_acl/tests/cli/overridden.yaml

```diff
@@ -0,0 +1,73 @@
+---
+- debug:
+      msg: "START Overridden ios_acl state for integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- include_tasks: _populate_config.yaml
+
+- block:
+    - name: Override device configuration of all interfaces with provided configuration
+      ios_acl: &overridden
+        config:
+          - afi: ipv4
+            acls:
+              - name: 110
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        ack: true
+                    source:
+                      address: 198.51.100.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    destination:
+                      address: 198.51.110.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: www
+                    dscp: ef
+                    ttl:
+                      eq: 10
+              - name: 150
+                aces:
+                  - grant: deny
+                    protocol_options:
+                      tcp:
+                        syn: true
+                    source:
+                      address: 198.51.100.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    destination:
+                      address: 198.51.110.0
+                      wildcard_bits: 0.0.0.255
+                      port_protocol:
+                        eq: telnet
+                    dscp: ef
+                    ttl:
+                      eq: 10
+        state: overridden
+      register: result
+
+    - assert:
```

Comment:
Please assert all the commands sent, before and after list of dicts.

Reply:
Commands fired are being checked through 3rd assert statement and same as above